scripts/install

```diff
@@ -15,5 +15,6 @@ else
     PIP="pip"
 fi
 
+"$PIP" install -U "pip >= 20.2" setuptools wheel
```

Comment:
Install `wheel` before others to avoid warnings like:
```
Using legacy 'setup.py install' for autoflake, since package 'wheel' is not installed.
```